docs/system_requirements/index.sdoc

```diff
@@ -266,3 +266,28 @@ Zepyhr shall provide a framework mechanism for tracing low level system operatio
 USER_STORY: >>>
 As a Zephyr RTOS user, I want to be able to trace different OS operations.
 <<<
+
+[SECTION]
+TITLE: Queues
+
+[REQUIREMENT]
+UID: ZEP-SYRS-20
+STATUS: Draft
+TYPE: Non-Functional
+COMPONENT: Queues
+TITLE: Queue data structure
+STATEMENT: >>>
+The Zephyr RTOS shall implement a queue data structure to hold data items.
```

Comment:
As formulated here this is somehow a truism. What is the intention of this requirement?

Reply:
That is true that requirement makes no sense at this point, I will remove it. The intention was to make the data item more detailed and highlighted but if I would have done this it would have been more of a design item.